content/docs/gke/deploy/deploy-cli.md

```diff
@@ -73,7 +73,7 @@ Follow these steps to deploy Kubeflow:
     export PATH=$PATH:<path to kfctl in your kubeflow installation>
     export ZONE=<your target zone> #where the deployment will be created
 
-    export PROJECT=<your GCP project>
+    export PROJECT=<your GCP project id>
```

Comment:
Since this text is plain language rather than code, please change this from "id" to "ID".

Reply:
changed this to ID.